.storybook/theme.js

```diff
@@ -0,0 +1,36 @@
+import { create } from '@storybook/theming';
+
+export default create({
```

Comment:
We can use this base theme--the Storybook defaults--to make changes as we see fit. 

I dug through the official Storybook Storybook too, to see why their styling is much better, and there's quite a bit of code involved for those small deviations. Tinkering with this will provide immediate small benefit, though.